samtranslator/model/connector_profiles/profiles.json

```diff
@@ -790,6 +790,23 @@
             }
           }
         }
+      },
+      "AWS::Events::EventBus": {
+        "Type": "AWS_IAM_ROLE_MANAGED_POLICY",
+        "Properties": {
+          "SourcePolicy": true,
+          "AccessCategories": {
+            "Write": {
```

Comment:
Do we not have read permissions?

Reply:
I discussed briefly about this with Slava at the start and we were thinking that the general workflow for eventBuses doesn't really make sense to allow you to read from it because we just send events through the eventBus to wherever we want it to go. The other existing connector for Function -> EventBus also only allows for writing and not reading. Do you think there are any scenarios where we'd want to be able to read from it?

Reply:
Yea I agree with that, since its just routing events there really is no need to read. Thanks for the explanation.